src/firefly/js/ui/DropDownMenu.jsx

```diff
@@ -81,5 +81,6 @@ DropDownMenuWrapper.propTypes= {
     visible : PropTypes.bool,
     x : PropTypes.number.isRequired,
     y : PropTypes.number.isRequired,
-    content : PropTypes.object.isRequired
+    content : PropTypes.object.isRequired,
+    zIndex : PropTypes.number
 };
```

Comment:
Everything looks good to me. The comments I made are all small things.  I also learned how to use reduce from reviewing Trey's code.  I have a GAIN.!!!